src/pilot_simulation.jl

```diff
@@ -279,23 +279,25 @@ slice_output_writer =
 ##### Horizontal averages output writer
 #####
 
+Hz = model.grid.Hz
+
 profiles = Dict(
-    "u"  => model ->  Up(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "v"  => model ->  Vp(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "w"  => model ->  Wp(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "T"  => model ->  Tp(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "S"  => model ->  Sp(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "ν"  => model ->  νp(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "κT" => model -> κTp(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "κS" => model -> κSp(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "uu" => model ->  uu(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "vv" => model ->  vv(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "ww" => model ->  ww(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "uv" => model ->  uv(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "uw" => model ->  uw(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "vw" => model ->  vw(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "wT" => model ->  wT(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "wS" => model ->  wS(model)[1+model.grid.Hz:end-model.grid.Hz]
+    "u"  => model ->  Up(model)[1+Hz:end-Hz],
```

Comment:
Could be nice to save the full profiles including halo regions. I hope to have an LES model with diffusivities prescribed on boundaries, which means the gradients of fields across boundaries will be meaningful (and that information is encoded in halos).

Reply:
True. Would be easy to add the option to `NetCDFOutputWriter`. I opened an issue about this: https://github.com/climate-machine/Oceananigans.jl/issues/683